README.md

```diff
@@ -27,6 +27,13 @@
 - Meteor Version: 2.1
 
 ### WIP
+#### Parties
+- Demo: N/A
+- Repository: [meteor/examples/parties](./parties)
+- Why: One of the original Meteor examples
```

Comment:
The idea here is to explain for the reader why this example is relevant, what features it is exemplifying or which technologies it is using how to use on Meteor projects, etc.

Could you elaborate this Why in this model?

Reply:
Updated